lib/immobilienscout/api/attachment.rb

```diff
@@ -15,6 +15,16 @@ def add(is24_id, binary_file, metadata)
           parsed_response
         end
 
+        def order(is24_id, params)
```

Comment:
would name it `put_order`  to make clear that one is **changing** the order.  If it just reads `order` one might easily think one would just get back the order of attachments.